frontend/src/components/placementVisuals/PlacementSamplesTable.tsx

```diff
@@ -147,13 +146,20 @@ const PlacementSamplesTable = ({ parentContainerName }: PlacementSamplesTablePro
                 dataIndex: 'containerName',
                 key: 'containerName',
             })
-            if (parentContainer.name !== null) {
+            if (parentContainer.getName() !== null) {
                 columns.push({
                     title: 'Src Coords',
                     dataIndex: 'coordinates',
                     key: 'coordinates',
+                    width: 120,
                 })
             }
+            columns.push({
+                title: 'Count',
```

Comment:
Count is too vague to my taste. I would use the longer dst count. or placement count or dest. #